app/instance-initializers/ember-i18n.js

```diff
@@ -14,6 +14,7 @@ export default {
     const key = 'service:i18n';
     const i18n = instance.lookup ? instance.lookup(key) : instance.container.lookup(key);
     i18n.set('locale', defaultLocale);
+    i18n.set('allowLocaleOverride', (ENV.i18n || {}).allowLocaleOverride);
```

Comment:
Except the service is in `addon/` and the config is in `app/`. Hmm. I don't know that there's a way to reference `ENV` from the service directly :(